spec/workers/articles/publish_worker_spec.rb

```diff
@@ -89,6 +89,19 @@
       end.to change(Notification, :count).by(2)
     end
 
+     it "does not create a notification for each users with new-post-notification setting = false" do
```

Comment:
@lightalloy  can I ask your advice on a workable approach for this test?  testing for `:upsert_all` or other `Notifications::` methods doesn't seem to work.